wcomponents-lde/pom.xml

```diff
@@ -43,8 +43,15 @@
 			<artifactId>junit</artifactId>
 		</dependency>
 
```

Comment:
This change was in line with wcomponents-examples. I am not really keen on having a dummy XSLT. We have a process which relies on an XSLT processor and we build XSLT so why not use the one we already have?